src/schema/v2/me/conversation/conversationEventConnection/conversationMessageEvents.ts

```diff
@@ -0,0 +1,67 @@
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { get } from "lodash"
+
+const lastMessageId = (conversation) => {
+  return get(conversation, "_embedded.last_message.id")
```

Comment:
Why use `get` here?

Reply:
this is copied from our old `messagesConnection` so there are definite opportunities for refactoring.